app/controllers/generic_object_definition_controller.rb

```diff
@@ -23,6 +24,30 @@ def self.model
     GenericObjectDefinition
   end
 
+  def button
+    javascript_redirect(
+      case params[:pressed]
```

Comment:
Missing a `_delete` case?

Reply:
I plan to implement `Delete` in a separate PR where I will use the `Delete` API.
Most probably it can be called directly from the Toolbar using -
```rb
:data => {'function' => 'sendDataWithRx',}
```

Reply:
Aah.. makes sense :) it's a bit confusing to add the buttons here if they don't work, but..